### PR TITLE
fix broken link between v2 and v1

### PR DESCRIPTION
**Pull request by prince-chrismc**
image

### CODE REVIEW

File: conan_v2.rst [hunk 1]
```diff
@@ -20,11 +22,11 @@ If you want to migrate to 2.0, there are several things you will need to change:
...
   - **General changes** not related to the recipes nor the Conan commands, "build profiles", lowercase references... etc.
 
-If you are looking for precompiled binaries, there is a very short list in a separate remote which can be added (please, check the `Conan 2.0 documentation <https://docs.conan.io/en/2.0/index.html>`_ for more information)
 
-    .. code-block:: bash
+.. notice::
 
-      $ conan remote add conanv2 https://conanv2beta.jfrog.io/artifactory/api/conan/conan --index 0
+    If you are looking for precompiled binaries, please, check the
+    `Conan 2.0 documentation <https://docs.conan.io/2/index.html>`_ for more information.
```
Comment (czoido) [inline conan_v2.rst:29]: Maybe we can replace this with a note explaining that we already have lots of recipes prepared for v2, some of them with generated binaries, in conancenter and pointing to the list of packages that already have binaries available ?

Comment (prince-chrismc) [reply conan_v2.rst:29]: That sounds like a much better idea :) 

EDIT: after looking into it, I am not sure we even have that list

https://conan.io/cci-v2.html
https://github.com/conan-io/conan-center-index/discussions/16196

> List last updated 2023-03-03 14:17 UTC

I think that's less helpful

**Comment by prince-chrismc** [comment]
I found two issues testing locally #3101  and #3100  🙊

**Review by memsharded — APPROVED** [review]

**Review by RubenRBS — APPROVED** [review]

File: conan_v2.rst [hunk 2]
```diff
@@ -4,12 +4,14 @@
 Conan migration guide to 2.0
 ============================
 
-Conan 2.0.0 `is already released <https://pypi.org/project/conan/#history>`_, you can
-install the latest Conan version from PyPI doing:
+.. tip::
 
-    .. code-block:: bash
+    Conan 2.0.0 `is already released <https://pypi.org/project/conan/#history>`_, you can
```
Comment (czoido) [inline conan_v2.rst:9]:
```suggestion
    Conan 2.0 `is already released <https://pypi.org/project/conan/#history>`_, you can
```

**Pull request merged**